flask_restplus_patched/parameters.py

```diff
@@ -1,9 +1,14 @@
 # encoding: utf-8
 # pylint: disable=missing-docstring
+import logging
+
 from six import itervalues
 
 from flask_marshmallow import Schema, base_fields
-from marshmallow import validate
+from marshmallow import validate, validates_schema
+from marshmallow.exceptions import ValidationError
```

Comment:
1. It seems that `validate` is not used, so it can be removed
2. `ValidationError` is exported to the main `marshmallow` scope, so you can rewrite these two lines into one:

```python
from marshmallow import validates_schema, ValidationError
```